packages/service/vivo.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"io"
+	"log"
+	"net/http"
+)
+
+func copyHeader(dst, src http.Header) {
+	for k, vv := range src {
+		for _, v := range vv {
+			dst.Add(k, v)
+		}
+	}
+}
+
+func notFound(w http.ResponseWriter) {
+	w.WriteHeader(404)
+	io.WriteString(w, "404 page not found")
+}
+
+func VivoListen(frontendStaticDir string) {
+	fs := http.FileServer(http.Dir(frontendStaticDir))
+	http.Handle("/", fs)
+
+	http.HandleFunc("/vivo/", func(w http.ResponseWriter, req *http.Request) {
+		if req.Method != "GET" {
+			notFound(w)
+			return
+		}
+
+		switch req.URL.Path {
+		case "/vivo/logs":
+		case "/vivo/metrics":
+		case "/vivo/traces":
+		default:
+			notFound(w)
+			return
+		}
+
+		client := &http.Client{}
+		//http: Request.RequestURI can't be set in client requests.
+		//http://golang.org/src/pkg/net/http/client.go
+		req.RequestURI = ""
+		req.Host = "127.0.0.1"
```

Comment:
We probably should make this configurable, e.g. for multi-NIC environments but not necessary for now.

Reply:
Not sure I understand how this could be useful in containers, but it can be useful to test the proxy in development if/when fluent-bit is running in another machine